### ISSUE TITLE
enh-ruby-use-ruby-mode-show-parens-config messes up deep indenting

### ISSUE THREAD

**Issue by muirrn** [zenspider/enhanced-ruby-mode#139]
Without `enh-ruby-use-ruby-mode-show-parens-config` I get this:

```
some_method({
  :foo => "bar",
})
```
But after turning on `enh-ruby-use-ruby-mode-show-parens-config` I get this:

```
some_method({
              :foo => "bar",
            })
```

**Comment by zenspider** [comment]
Assuming all the indentation issues that come up can get tests written for them and they can get driven out of the system, is there any reason to keep smie support around? Seems like it should just be deprecated and eventually removed, no? Objections?

**Comment by muirrn** [comment]
See #140 where I removed the dependency on ruby-mode (so smie no longer interferes with indentation).

**Comment by zenspider** [comment]
This is fixed, yes?

**Comment by muirrn** [comment]
Yes, thank you.

Fixed in e592beadcb2ddd55c62e8e831a038d02aad5ee2c for future reference.
